templates/macros.twig

```diff
@@ -35,3 +35,34 @@
 	{% import _self as macros %}
 	{{ macros.icon_stack('fa-square fa-stack-2x', "#{icon} fa-stack-1x fa-inverse", class , tooltip ) }}
 {% endmacro %}
+
+{# Create a competition entry #}
+{# params: competition name, due date, url of all the submissions, embed player URL
+   original description, list of rules #}
+{% macro competition_entry(name, id, due_date, group_url, player_url, desc, rules, extras = '') %}
```

Comment:
This macro is wrong. It can't be applied to currently existing competitions. If you ask me this adds another level of complexity to the pages and makes it hard to add new entries. This is only done once each year so I don't think this macro is really helping out.